controllers/item.js

```diff
@@ -19,11 +19,23 @@ item.withFields = (queryBuilder) => {
   // Category
     .leftJoin('category', 'item.categoryID', 'category.categoryID')
     .select('category.name as category')
+
+  // Mapping between query param fields and database query column names
+  const filterParams = {
+    brand: 'brand.brandID',
+    model: 'model.modelID',
+    category: 'category.categoryID'
+  }
```

Comment:
Would it make sense to name the params `brandID`, `modelID`, etc instead since we have to send the ids?